src/components/RandomBeer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Header from './Header'
+
+export default class RandomBeer extends Component {
+
+    state = {
+        randomBeer: ''
+    }
+
+    componentDidMount() {
+        fetch(`https://ih-beers-api2.herokuapp.com/beers/random`)
+          .then(res => res.json())
+          .then(result => {
+            console.log(result)
+            this.setState({randomBeer: result})
+          })
+          .catch(e => console.log(e))
+    }
+
+    render() {
```

Comment:
` const {name, tagline, image_url, first_brewed, attenuation_level, description, contributed_by} = this.state.randomBeer` Same idea for line 24-30.